apps/opik-backend/src/main/java/com/comet/opik/domain/FeedbackScoreDAO.java

```diff
@@ -433,6 +452,34 @@ public Mono<List<String>> getExperimentsFeedbackScoreNames(List<UUID> experiment
         });
     }
 
+    @Override
+    @WithSpan
+    public Mono<List<String>> getProjectsFeedbackScoreNames(List<UUID> projectIds) {
+        return asyncTemplate.nonTransaction(connection -> {
+
+            ST template = new ST(SELECT_PROJECTS_FEEDBACK_SCORE_NAMES);
+
+            if (CollectionUtils.isNotEmpty(projectIds)) {
+                template.add("project_ids", projectIds);
+            }
+
+            var statement = connection.createStatement(template.render());
+
+            if (CollectionUtils.isNotEmpty(projectIds)) {
+                template.add("project_ids", projectIds);
+            }
+
+            if (CollectionUtils.isNotEmpty(projectIds)) {
+                statement.bind("project_ids", projectIds.toArray(UUID[]::new));
+            }
+
+            return makeMonoContextAware(bindWorkspaceIdToMono(statement))
+                    .flatMapMany(result -> result.map((row, rowMetadata) -> row.get("name", String.class)))
+                    .distinct()
```

Comment:
Minor: distinct already happens in the DAO, but ok to reinforce it here, but watch for performance as unordered etc.